Project.toml

```diff
@@ -11,8 +11,7 @@ Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-BinDeps = "≥ 0.4.0"
-CMake = "1"
-Parameters = "≥ 0.5.0"
-julia = "≥ 0.7.0"
-
+BinDeps = "0.4, 0.8, 1"
+CMake = "1, 1.2"
```

Comment:
I think `CMake = "1"` is still the right thing to do--that will allow any version which is compatible with CMake 1.0 according to semver, meaning any version 1.x.

Reply:
Fixed in https://github.com/JuliaPackaging/CMakeWrapper.jl/pull/24/commits/ffb8bf179cdb2442b10b89de3f1ac4bfe2677b20